src/SIL.XForge.Scripture/appsettings.json

```diff
@@ -5,8 +5,11 @@
     }
   },
   "AllowedHosts": "*",
-  "System": {
-    "Hostname": "beta.scriptureforge.org"
+  "Site": {
```

Comment:
I know this is still in process, but there's a couple of things to look at here.
I think the System section was removed by accident and the SmtpServer probably still needs an update.